pilot/pkg/proxy/envoy/v2/monitoring.go

```diff
@@ -16,44 +16,43 @@ package v2
 import (
 	"google.golang.org/grpc/codes"
 
-	"istio.io/istio/pilot/pkg/monitoring"
 	"istio.io/istio/pkg/mcp/status"
+	"istio.io/pkg/monitoring"
 )
 
 var (
-	errTag     = monitoring.MustCreateTag("err")
-	clusterTag = monitoring.MustCreateTag("cluster")
-	nodeTag    = monitoring.MustCreateTag("node")
-	typeTag    = monitoring.MustCreateTag("type")
+	errTag     = monitoring.MustCreateLabel("err")
+	clusterTag = monitoring.MustCreateLabel("cluster")
+	nodeTag    = monitoring.MustCreateLabel("node")
+	typeTag    = monitoring.MustCreateLabel("type")
 
 	cdsReject = monitoring.NewGauge(
 		"pilot_xds_cds_reject",
 		"Pilot rejected CSD configs.",
-		nodeTag, errTag,
```

Comment:
does this not need the WithLabels like the other?

Reply:
it should